### DISCUSSION TITLE
Where to Find a Good Starter Config?

### DISCUSSION THREAD

**Discussion by Christopher-Hayes** [wezterm/wezterm#1179] [Q&A] [answered]
Anyone have a gist or somewhere for starter config files to get me started working with Wezterm? Over time I'll go through the documentation and set specific configs, but looking for a config to jumpstart working with Wezterm. I'm on MacOS, so if anyone has a Mac-oriented one, all the better! I find it's easier to take a full config and remove/change things, than to start from scratch.

**Comment by wez** [comment, marked as answer]
Take a look at: https://github.com/wez/wezterm/discussions/628
and https://github.com/wez/wezterm/issues/548

**Reply by Christopher-Hayes** [reply]
Thanks!
